src/api/users/test/e2e/users.test.js

```diff
@@ -1,203 +1,81 @@
-  //   expect(response.status).toBe(200);
-  //   expect(response.body.length).toBe(0);
-  //   expect(response.body).toEqual([]);
-  // });
-
-  // test.skip('Accepted - Test negative perPage parameter, positive page parameter', async () => {
-  //   const galileo = await createUser();
-  //   expect(galileo.response.status).toBe(201);
-
-  //   const response = await getUsersParams(-20, 1);
-  //   expect(response.status).toBe(400);
-  // });
-
-  // test.skip('Accepted - Test negative perPage parameter, negative page parameter', async () => {
-  //   const galileo = await createUser();
-  //   expect(galileo.response.status).toBe(201);
-
-  //   const response = await getUsersParams(-20, -1);
-  //   expect(response.status).toBe(400);
-  // });
-
-  // test.skip('Accepted - Test positive perPage parameter, negative page parameter', async () => {
-  //   const galileo = await createUser();
-  //   expect(galileo.response.status).toBe(201);
-
-  //   const response = await getUsersParams(20, -1);
-  //   expect(response.status).toBe(400);
-  // });
-
-  // test.skip('Accepted - Test page and per_page parameters', async () => {
-  //   await createUser(defaultUsers.galileo());
-  //   await createUser(defaultUsers.carl());
-
-  //   // create 2 users, request 1 user per page and page 2
-  //   const response = await getUsersParams(1, 2);
-  //   expect(response.body.length).toBe(1);
-  //   expect(response.body).toEqual([defaultUsers.galileo()]);
-  // });
-
-  // test('Accepted - Test parameterized get, post one user, ask for five, receive one user', async () => {
-  //   const galileo = await createUser();
-  //   expect(galileo.response.status).toBe(201);
-
-  //   // request 5 users per page on page 2
-  //   const response = await getUsersParams(5, 2);
-  //   expect(response.status).toBe(200);
-  //   expect(response.body.length).toBe(1);
-  //   expect(response.body).toEqual([defaultUsers.galileo()]);
-  // });
-
-  test('Accepted - Get one user', async () => {
-    const { user } = await createUser();
-    const response = await getUser(user.id);
-    expect(response.statusCode).toBe(200);
-    expect(response.body).toEqual(defaultUsers.galileo());
+  it('should return 404 if the user does not exist', async () => {
+    const response = await getUser({ email: 'nonexistent@email.com' });
+    expect(response.status).toBe(404);
   });
 
-  test('Rejected - Get a user which does not exist', async () => {
-    const response = await getUser(createUserHash('no-such-user@nowhere.com'));
-    expect(response.statusCode).toBe(404);
+  it('should 400, rejected by negative per_page', async () => {
+    // start_after does not matter, middleware should reject regardless
+    const response = await getUsersPaginated('?per_page=-5');
+    const users = await response.json();
+    expect(response.status).toBe(400);
+    expect(users.validation.query.message).toBe('"per_page" must be greater than or equal to 1');
```

Comment:
The issue with these error messages is that if Celebrate changes how they word something, our tests will start failing, even though we still pass functionally.

In other words, never depend on the wording of error messages.  Remove this line, and only do the 400 check on line 69.